src/mon/MonCommands.h

```diff
@@ -869,6 +869,31 @@ COMMAND("osd pool get-quota " \
         "name=pool,type=CephPoolname ",
         "obtain object or byte limits for pool",
         "osd", "r", "cli,rest")
+COMMAND("osd pool application enable " \
```

Comment:
IMO the generic and the 'rbd pool-create' (or whatever) is sufficient; we don't need a third.